docs/develop/dotnet/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+id: index
+title: Temporal .NET SDK feature guidance
+sidebar_label: .NET SDK
+description: How to use Temporal features using the Temporal .NET SDK.
+toc_max_heading_level: 4
+keywords:
+  - dotnet
+  - sdk
+  - feature guidance
+tags:
+  - dotnet
+  - sdk
+  - feature guidance
+---
+
+<!-- .NET SDK feature guidance landing page-->
```

Comment:
This is the first ever feature guidance landing page, so unsure what to add here. I am assuming I follow the dev-guide landing pages?